src/main/java/de/zalando/aruha/nakadi/service/EventStream.java

```diff
@@ -42,15 +40,16 @@ public EventStream(final EventConsumer eventConsumer, final OutputStream outputS
 
     public void streamEvents() {
         try {
+            outputStream.flush();
```

Comment:
I thought you will do that in controller, seems to be a more suitable place from logic separation point of view.
What do you think?